code/modules/mob/living/carbon/human/species/_species.dm

```diff
@@ -471,7 +471,7 @@
 	if(target.check_block())
 		target.visible_message("<span class='warning'>[target] blocks [user]'s disarm attempt!</span>")
 		return FALSE
-	if(attacker_style && attacker_style.disarm_act(user, target))
+	if(attacker_style && attacker_style.disarm_act(user, target) == TRUE)
```

Comment:
Shouldn't this work without the `== TRUE`?

Reply:
Indeed so. Was some left over code of me trying to make this one return multiple values.